packages/wdio-browserstack-service/src/service.js

```diff
@@ -139,7 +140,7 @@ export default class BrowserstackService {
     }
 
     async _printSessionURL() {
-        const capabilities = global.browser.capabilities
+        const capabilities = global.browser.capabilities || {}
```

Comment:
I don't think `global.browser.capabilities` can ever be undefined

Reply:
Are you sure, because this is the exact exception I encountered when using multiremote?

Reply:
ah right, for multi-remote this is indeed undefined. Then let's add a comment to make sure people reading this understand.

Reply:
Okay, but there's 6 places where we assign default value for this in the entire source code. I wonder if it's better to actually assign a default value for `global.browser.capabilities`? If it's possible, where we should do that

Reply:
@hieuxlu that is a good point. The problem is that some of the service functionalities don't work. This patch just makes it not break. I suggest to create an issue for every package where this happens and move on with the current solution. 

WDYT?

Reply:
Agree, added the comments